prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,50 @@
+let number1;
+let number2;
+
+number1 = 10;
+number2 = 20;
```

Comment:
Rather than separating these, I would do the assignment alongside the declaration

Reply:
Added a new commit combining the assignment and declaration of these variables.